test/gpc.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert')
+
+const {
+    GPC_HEADER_PRIORITY
+} = require('../lib/rulePriorities')
+
+const {
+    generateGPCheaderRules
+} = require('../lib/gpc')
+
+const baseExtensionConfig = {
+    features: {
+        gpc: {
+            state: 'enabled',
+            exceptions: [{
+                domain: 'content-blocking1.example',
```

Comment:
Maybe change the domain so it's clear it's unrelated to contentBlocking allowlist? (Same below.)
```suggestion
                domain: 'exception1.example',
```